### ISSUE TITLE
Deep Learning for the Single-image Super Resolution Reconstruction of QSM Maps

### ISSUE THREAD

**Issue by Lewis-Kitchingman** [Lewis-Kitchingman/MIC-HACK-2025#7]
**Project Leaders:** Stefano Zappala (zappalas@cardiff.ac.uk), Eleonora Patitucci (patituccie@cardiff.ac.uk)

Long acquisitions or high-field systems are required to produce quantitative susceptibility maps (QSM) containing enough details to reliably capture the vascular network and estimate accurate values of oxygen saturation. Single-image Super-resolution Reconstruction (SISR) methods based on deep learning architectures can help in the restoration of those small vessels (such as cortical veins) that cannot be otherwise captured with limited time or field strength. 

**Ideal Participant Characteristics:** Experience in Python programming and image analysis. Experience in carrying out a deep learning experiment is a plus but not required. Knowledge of QSM imaging would be ideal for final interpretations of results. 

**Resources:**  
Libraries/Software: Tensor-flow / PyTorch / SEPIA (MATLAB). 
Data: Public database WAND. 
Hardware: No specific hardware needed as access to CUBRIC cluster will be available (otherwise, Google Colab). 

**Project Plan:**  

**First Task:** 
Goal: Identify two / three deep learning architectures ideal for the SISR of high spatial frequency components (that is, veins). 
Steps: Literature review + identification of freely available code. 

Expertise: knowledge of the basics of a deep learning experimentation (data preprocessing, training, inference, etc). 

**Second Task:** 
Goal:  Map generation. 
Steps: Basic neuroimaging preprocessing: NIFTI conversion, skull stripping, etc. Extraction of QSM maps with SEPIA.  
Expertise: Programming in Python and basic image analysis. 

**Third Task:** 
Goal: Training of the chosen architectures. 
Steps: Prepare data for training (patching, normalisation, etc). Implementation of suitable cross validation. Inference. 
Expertise: Programming in Python, basic knowledge of tensorflow / PyTorch. 

**Fourth Task:** 
Goal: Identify best performing architecture. 
Steps: Compare performance metrics. Explainable AI to understand what is happening under the bonnet. 
Expertise: Programming in Python, knowledge of the behaviour of common deep learning architectures (from task 1).
